command/sidechain/helper.go

```diff
@@ -33,15 +32,32 @@ func CheckIfDirectoryExist(dir string) error {
 	return nil
 }
 
-func GetAccountFromDir(dir string) (*wallet.Account, error) {
-	secretsManager, err := secretsHelper.SetupLocalSecretsManager(dir)
+func ValidateSecretFlags(dataDir, config string) error {
+	if config == "" {
```

Comment:
Should you validate provided `config` value (at least you could check if the directory exists)?

Reply:
That is done while creating secret manager. Proper message will be returned.

Reply:
As far as I can see, it is going to try and read a config file here:
https://github.com/0xPolygon/polygon-edge/blob/develop/secrets/config.go#L29-L33. Maybe we could add there explicit check if file doesn't exist and return a proper error message (not sure if `os.ReadFile` is going to provide a meaningful error message)?